dali/python/nvidia/dali/plugin/base_iterator.py

```diff
@@ -72,9 +72,15 @@ class _DaliBaseIterator(object):
                 samples that iterator needs to deal with. It allows `last_batch_policy` to be
                 PARTIAL or DROP, if FILL is used it is changed to PARTIAL. Sets `last_batch_padded`
                 accordingly to the reader's configuration (`pad_last_batch` reader argument)
-    auto_reset : bool, optional, default = False
-                Whether the iterator resets itself for the next epoch
-                or it requires reset() to be called separately.
+    auto_reset : string or bool, optional, default = False
+                Whether the iterator resets itself for the next epoch or it requires reset() to be called separately.
```

Comment:
```suggestion
                Whether the iterator resets itself for the next epoch or it requires reset() to be called explicitly.
```

Reply:
Same comment applies to all copies of this documentation string

Reply:
Fixed